app/Http/Controllers/PaymentRequestController.php

```
@@ -154,11 +155,18 @@ public function store(PaymentRequestStore $request)
+                ->first();
+            if ($nextFlow->user_id) {
+                User::find($nextFlow->user_id)->notify(new RequestTransactionNotification(auth()->user()->token, $nextFlow));
+            }
             DB::commit();
-            foreach ($request->attachment_file_names as $file) {
-                $path = 'prf/'.$paymentRequest->id.'/'.$file;
-                Storage::move('temp/'.$file, $path);
+            if ($request->attachment_file_names) {
+                foreach ($request->attachment_file_names as $file) {
+                    $path = 'prf/'.$paymentRequest->id.'/'.$file;
+                    Storage::move('temp/'.$file, $path);
```

PHPMD (2.15.0)

Avoid using static access to class '\Storage' in method 'store'. (undefined)

(StaticAccess)